src/vcpkg/commands.cpp

```diff
@@ -41,13 +42,16 @@ namespace vcpkg::Commands
     {
         static const Version::VersionCommand version{};
         static const Contact::ContactCommand contact{};
+        static const InitRegistry::InitRegistryCommand initRegistry{};
 #if defined(_WIN32)
         static const UploadMetrics::UploadMetricsCommand upload_metrics{};
 #endif // defined(_WIN32)
 
         static std::vector<PackageNameAndFunction<const BasicCommand*>> t = {
             {"version", &version},
             {"contact", &contact},
+            {"init-registry", &initRegistry},
```

Comment:
```suggestion
            {"x-init-registry", &initRegistry},
```

this should be an experimental command.